sys/prebuilt-bindings/geos_3.10.rs

```diff
@@ -0,0 +1,2717 @@
+/* automatically generated by rust-bindgen 0.60.1 */
+
+pub const GEOS_VERSION_MAJOR: u32 = 3;
+pub const GEOS_VERSION_MINOR: u32 = 10;
+pub const GEOS_VERSION_PATCH: u32 = 3;
+pub const GEOS_VERSION: &[u8; 7usize] = b"3.10.3\0";
```

Comment:
Should be `&str` (without the `\0`).

Reply:
Note: these weren't in the previous bindings, but based on other bindings to GEOS, I think that it is a good idea to keep them.  Some of these consts can probably get excluded though, they aren't useful here: `GEOS_JTS_PORT`, `GEOS_CAPI_FIRST_INTERFACE`, `GEOS_CAPI_LAST_INTERFACE`.

Reply:
It's pretty bad to keep the original C string with the `\0` at when it's unneeded.